spec/requests/api/picture_spec.rb

```diff
@@ -6,9 +6,13 @@
 # - Query picture and image_href of service_requests   /api/service_requests/:id?attributes=picture,picture.image_href
 #
 describe "Pictures" do
+  # Valid base64
+  let(:content) do
+    "aW1hZ2U="
+  end
   let(:dialog1)  { FactoryGirl.create(:dialog, :label => "ServiceDialog1") }
   let(:ra1)      { FactoryGirl.create(:resource_action, :action => "Provision", :dialog => dialog1) }
-  let(:picture)  { FactoryGirl.create(:picture, :extension => "jpg") }
+  let(:picture)  { FactoryGirl.create(:picture, :extension => "jpg", :content => content) }
```

Comment:
Doesn't this give the picture a with un-decoded content? Which I know probably doesn't matter, but it does help the next person looking at this :smile: